ifrs17/Import/Importers.ipynb

```diff
@@ -1368,13 +1368,17 @@
                 "\n                if(values.Length == 0 && !parsingStorage.MandatoryAocSteps.Contains(new AocStep(aocType, novelty))) return null;",
                 "\n            }",
                 "\n            ",
+                "\n            var accidentYearInput = (datarow.Field<string>(nameof(RawVariable.AccidentYear)));",
+                "\n            if(hasAccidentYearColumn) ",
```

Comment:
first thing, check if there is the column

Reply:
I modified the structure a bit, but still the vars are declared before the if statement for the `hasAccidentYearColumn`, does it look okay?